app/serializers/v1/subscription_serializer.rb

```diff
@@ -53,7 +53,7 @@ def entitlements
     def plan
       ::V1::PlanSerializer.new(
         model.plan,
-        includes: %i[charges usage_thresholds taxes minimum_commitment]
+        includes: %i[charges fixed_charges usage_thresholds taxes minimum_commitment]
```

Comment:
do we really need it on the sub? We add a new query on the sub serializer and subscriptions endpoints are really slow because of loading everything